spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultPublisherFactory.java

```diff
@@ -71,38 +73,41 @@ public DefaultPublisherFactory(GcpProjectIdProvider projectIdProvider) {
 	}
 
 	/**
-	 * @param executorProvider provides the executor to be used by the publisher, useful to specify
-	 *                         the number of threads to be used by each executor
+	 * Sets the provider for the executor that will be used by the publisher.
+	 *
```

Comment:
This fails our Spring code style:

> Use an imperative style (i.e. Return and not Returns) for the first sentence.
No blank lines between the description and the parameter descriptions.

And no `<p>`, please, in method JavaDocs!

https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

Reply:
The link you pasted says `<p>` should be used for multiple paragraphs on methods.

```
If the description is defined with multiple paragraphs, start each of them with <p>.
```

Am I missing anything?

Reply:
Well, that "please" was really from me: I don't see reason to make separate paragraphs just from two sentences.

You are failing to follow other rules though 😄

Reply:
You'd be surprised by the amount of different opinions on that "one-liner method description" rule. :P

In any case, I can change that if you want.

Reply:
Yes, please. I see reason in a new paragraph if we really start to talk about a new matter, like we do writing the book. But if the second sentence is  a continuation of the first one, that isn't read clearly.

Let's just follow what SF dictates and don't pay attention to other opinions which just would waste our time.